source/onboard/mattermost-limits.rst

```diff
@@ -0,0 +1,46 @@
+Workspace limits
+================
+
+There may be limits applied to your Mattermost Cloud workspace, based on the type of subscription plan you have. These limits do not apply to self-hosted deployments.
```

Comment:
Propose adding a paragraph/sentence somewhere that there are additional features that paid subscriptions unlocked, with a complete list available in https://mattermost.com/pricing/

(Want to avoid confusion where someone thinks apart from these limits, all other features are available in the Cloud Starter plan)

Reply:
Good call. I'll add it.